packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -97,51 +106,18 @@ export class DataSyncMongoDBDataProvider<Type = any> extends MongoDBDataProvider
       filter = {};
     }
 
-    filter._deleted = { ne: true };
+    filter[DataSyncFieldNames.deleted] = { ne: true };
 
     return super.count(filter);
   }
 
-  public sync(lastSync: string, context: GraphbackContext, filter?: any): Promise<Type[]> {
+  public sync(lastSync: number, context: GraphbackContext, filter?: any): Promise<Type[]> {
 
     return super.findBy({
       ...filter,
-      updatedAt: {
-        gt: lastSync
+      [DataSyncFieldNames.lastUpdatedAt]: {
+        ge: lastSync
```

Comment:
Because `lastSync` is not a number, this query is not working (tested from the playground).